.gitignore

```diff
@@ -20,3 +20,7 @@ assets/jsconfig.json
 # VS Code
 /.vscode/**
 !/.vscode/cspell.json
+
+# .DS_Store 
+.DS_Store 
+*/.DS_Store 
```

Comment:
I'd rather we add this separately, which is what I've done via #1238 (FYI). So please remove this change from this PR

Reply:
got it.

Reply:
#1238 has been merged, so if you rebase, you'll be able to pick it up. Thanks